src/pages/AssignServerRoles.js

```diff
@@ -652,58 +627,16 @@ class AssignServerRoles extends BaseWizardPage {
       });
 
     // get manually added servers
-    fetch(getAppConfig('shimurl') + '/api/v1/server?source=manual')
-      .then(response => response.json())
+    fetchJson('/api/v1/server?source=manual')
       .then((responseData) => {
         if (responseData.length > 0) {
           this.setState({serversAddedManually: responseData});
         }
       });
   }
 
-  getSavedDiscoveredServers() {
-    return (
-      fetch(getAppConfig('shimurl') + '/api/v1/server?source=sm,ov')
-        .then((response) => this.checkResponse(response))
-        .then(response => response.json())
-    );
-  }
-
-  saveDiscoveredServers(resultServers) {
-    return (
-      fetch(getAppConfig('shimurl') + '/api/v1/server', {
-        method: 'POST',
-        headers: {
-          'Accept': 'application/json, text/plain, */*',
-          'Content-Type': 'application/json'
-        },
-        body: JSON.stringify(resultServers)
-      })
-        .then((response) => this.checkResponse(response))
-    );
-  }
-
-  updateDiscoveredServer(server) {
-    return (
-      fetch(getAppConfig('shimurl') + '/api/v1/server', {
-        method: 'PUT',
-        headers: {
-          'Accept': 'application/json, text/plain, */*',
-          'Content-Type': 'application/json'
-        },
-        body: JSON.stringify(server)
-      })
-        .then((response) => this.checkResponse(response))
-    );
-  }
-
```

Comment:
it still call this updateDiscoveredServer at line 938

Reply:
Thanks for catching that.  Will address it.